api/backend/ecs/resource_manager.go

```diff
@@ -61,11 +61,6 @@ func (r *ECSResourceManager) getResourceProvider(ecsEnvironmentID id.ECSEnvironm
 		return nil, false
 	}
 
-	if !pbool(instance.AgentConnected) {
-		r.logger.Errorf("Instance %s agent is disconnected", instanceID)
```

Comment:
we should still log

Reply:
Okeydoke. I've got it at `Info` level right now - maybe `Debug` level is more appropriate...?